### PR TITLE
Anonymous Sessions

### PR DESCRIPTION
**Pull request by philpursglove**
For feedback. (I have no idea why the commit from @adrianbanks is showing as part of this PR...)

### CODE REVIEW

**Comment by alastairs** [comment]
Probably that commit's not been merged to `production`. I'll take a look at the graph and see what's going on.

File: DDDEastAnglia/Views/Session/Details.cshtml [hunk 1]
```diff
@@ -5,25 +5,18 @@
...
+    <div>
+        @if (Model.SpeakerGravatarUrl != null)
+        {
+            <img src="@Model.SpeakerGravatarUrl" alt="@Model.SpeakerName"  />
+        }
+        <a href="@Url.Action("Details", "Speaker", new {id = Model.SpeakerId})">@(string.IsNullOrWhiteSpace(Model.SpeakerName) ? Model.SpeakerUserName : Model.SpeakerName)</a>
+    </div>
+}
 <div class="tweet-session">
-    @if (User.Identity.Name == Model.SpeakerUserName)
```
Comment (alastairs) [inline DDDEastAnglia/Views/Session/Details.cshtml:18]: Why did we delete this `if` statement?

**Comment by alastairs** [comment]
Yeah, looks like the branch is based off `staging` rather than `production`. If you rebase, @adrianbanks' merge should disappear.

File: DDDEastAnglia/Views/Session/Details.cshtml (hunk 1 above)
Comment (philpursglove) [inline DDDEastAnglia/Views/Session/Details.cshtml:18]: Because if sessions are going to be anonymous then speakers don't get to tweet 'Look at my session'

Comment (alastairs) [inline DDDEastAnglia/Views/Session/Details.cshtml:18]: ...but this is about anonymous voting?

**Comment by philpursglove** [comment]
Ahh, will rebase off production this afternoon

File: DDDEastAnglia/Views/Session/Details.cshtml (hunk 1 above)
Comment (philpursglove) [inline DDDEastAnglia/Views/Session/Details.cshtml:18]: I've changed the title to the more descriptive 'Anonymous sessions'

**Comment by philpursglove** [comment]
Rebased to production, @adrianbanks' commit has gone :-)

File: DDDEastAnglia/DataAccess/ConferenceLoader.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ public interface IConferenceLoader
 
     public class ConferenceLoader : IConferenceLoader
     {
-        private const string DefaultEventName = "DDDEA2014";
+        private const string DefaultEventName = "DDDEA2015";
```
Comment (alastairs) [inline DDDEastAnglia/DataAccess/ConferenceLoader.cs]: Is this going to screw up the production site at all? E.g., it's currently closed, will this change open it again?

**Comment by alastairs** [comment]
This looks ok to me. There is a lot of unrelated changes (whitespace, mostly), which would have been better split into a separate pull request for ease of diffing. I like to stick to the Single Responsibility Principle for pull requests, as it makes it much easier to deal with them.

(See also [10 Tips for Better Pull Requests](http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/))

File: DDDEastAnglia/DataAccess/ConferenceLoader.cs (hunk 2 above)
Comment (adrianbanks) [inline DDDEastAnglia/DataAccess/ConferenceLoader.cs]: It probably will, yes.

**Pull request closed without merging**